### PR TITLE
Badges-component

### PR DESCRIPTION
**Pull request by anshtripathi01**
**_Please Review my Badge Component_**
Badges
s

### CODE REVIEW

File: Badges/badges.css [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    padding: 0px 0px 4px 6px;
+    z-index: 1;
+
+}
+
+
+.home-icon{
+    margin-left: 25px;
+    font-size: 2rem;
+    color:tomato;
```
Comment (zabihshaik) [inline Badges/badges.css:69]: Better to use HEX values for a color and use variables for the colors.

**Review by zabihshaik — COMMENTED** [review]
Everything looks good. 
Just a suggestion is it will be better to use HEX values for a color and use variables for the colors.

**Comment by anshtripathi01** [comment]
Thank you zabih I look into it

File: Badges/badges.css [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+body{
+    box-sizing: border-box;
+    font-family: 'Roboto', sans-serif;
+}
+.badges-container{
+    display: flex;
+   justify-content: flex-start;
+   align-items: center;
+   box-shadow: 0px 1px 3px 0px rgba(0, 0 ,0,0.1);
+   width: 100vh;
```
Comment (shashikantmaurya0007) [inline Badges/badges.css:12]: you have put vh it should be vw

File: Badges/badges.css [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    height: 200px;
+}
+.avtar{
+    margin: 20px;
+}
+.badges {
+    width:20px;
+    height: 20px;
```
Comment (shashikantmaurya0007) [inline Badges/badges.css:20]: give value in rem or any relative value

**Review by shashikantmaurya0007 — COMMENTED** [review]
these minor thing you should take care in future .rest is good;)

**Pull request merged**